### PR TITLE
Fix `FutureIntoCoroutine` runtime binding

### PR DESCRIPTION
**Pull request by TimeEngineer**
# What has changed ?

- [ ] Does this PR affect the User ?

### CODE REVIEW

File: src/runtime.rs [hunk 1]
```diff
@@ -1,17 +1,18 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
 
+#![allow(dead_code)]
+
```
Comment (FirelightFlagboy) [inline src/runtime.rs:4]: Remove the code that ins't used

**Review by FirelightFlagboy — CHANGES_REQUESTED** [review]

File: src/runtime.rs (hunk 1 above)
Comment (TimeEngineer) [reply src/runtime.rs:4]: Ok

Comment (TimeEngineer) [reply src/runtime.rs]: Done

Comment (FirelightFlagboy) [reply src/runtime.rs]: Depends on how fast you review the said PR

**Review by FirelightFlagboy — APPROVED** [review]

**Pull request merged**